src/main/java/pw/kaboom/extras/modules/checks/DenyTorchLoad.java

```diff
@@ -0,0 +1,34 @@
+package pw.kaboom.extras.modules.checks;
+
+import org.bukkit.Material;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.BlockPlaceEvent;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.plugin.java.JavaPlugin;
+import pw.kaboom.extras.Main;
+
+public class DenyTorchLoad implements Listener {
+    private static final int limit = JavaPlugin.getPlugin(Main.class).getConfig().getInt("torchLimit");
+
+    @EventHandler
+    public void onBlockPlace(BlockPlaceEvent event) {

```

Comment:
This entire event listener will never work. The amount of torches in a player's inventory is in no way proportional to how many torches are in the world/chunk. This event will do nothing but confuse players.